src/bootstrap/index.js

```diff
@@ -57,11 +60,22 @@ const setup = (
     });
 
   if (options.port !== false)
-    middleware.listen(app, {
+    server = middleware.listen(app, {
       port: options.port || config.get("port"),
       host: options.host || config.get("host"),
     });
 
+  process.on("SIGTERM", () => {
+    debug("SIGTERM signal received: closing HTTP server");
+    if (server) {
+      server.close(() => {
```

Comment:
I think you can pass an `err` parameter to close so that we can see an error message if something goes wrong. 

Perhaps something like:

```
server.close((err) => {
  if (err) {
    debug(`Error closing HTTP server: ${err.message}`);
  } else {
    debug("HTTP server has been successfully closed.");
  }
});
```